routes/user.js

```diff
@@ -0,0 +1,137 @@
+const User = require('../schemas/User')
+
+const Oo = require('../schemas/Oo')
+const Feedback = require('../schemas/Feedback').Feedback
+
+module.exports = app => {
+  /**
+   * @api {get} /users User list
+   * @apiName User list
+   * @apiGroup User
+   *
+   * @apiSuccess {Object[]} users List of users
+   *
+   * @apiSuccessExample Success-Response:
+   *     HTTP/1.1 200 OK
+   *     {
+   *       "users": [
+   *            {
+   *                "id": 1,
+   *                "username": "Test",
+   *                "email": "test@test.com",
+   *                "oos": [],
+   *                "feedbacks": [],
+   *            }
+   *        ]
+   *     }
+   */
+  app.get('/users', (req, res) => {
+    User.findAll({
+      attributes: ['id', 'username', 'email'],
+      include: [
+        {
+          model: Oo,
+          attributes: ['id', 'name', 'description'],
+          through: {
+            attributes: [],
+          },
+        },
+        {
+          model: Feedback,
+          attributes: ['id', 'status', 'createdAt'],
+          include: [
+            {
+              model: Oo,
+              attributes: ['id', 'name', 'description'],
+              through: {
+                attributes: [],
+              },
+            },
+          ],
+        },
+      ],
+      order: [
+        ['id', 'ASC'],
+        [Oo, 'id', 'ASC'],
+        [Oo, 'createdAt', 'ASC'],
+      ],
+    }).then(users => {
+      res.send({ users })
```

Comment:
If no users, nothing is send ?

Reply:
Si aucun utilisateur n'est trouvé, le JSON retourné sera un tableau vide :  `{ "users": [] }`